k8s/postgresql/Makefile

```diff
@@ -1,98 +1,124 @@
-include ../app.Makefile
 include ../crd.Makefile
 include ../gcloud.Makefile
 include ../var.Makefile
 
+APP_ID ?= postgresql-managed-updates
 
-TAG ?= 9.6
+# The digest should be updated automatically, or tag <major>.<minor>.<path/build>
+# should be available on GCP Marketplace.
+# Use `.build/postgresql/VERSION` target to fill the RELEASE variable.
+POSTGRESQL_TAG ?= sha256:9aa2e8b294818b834ecf5bb83ce84bb5741038aef81650d0a6c0ddea0a0df897
 METRICS_EXPORTER_TAG ?= v0.5.1
-SRCTAG ?= 9.6-kubernetes
-IMAGE_POSTGRESQL ?= marketplace.gcr.io/google/postgresql9:$(SRCTAG)
-IMAGE_POSTGRESQL_EXPORTER ?= marketplace.gcr.io/google/postgresql9:exporter
-$(info ---- TAG = $(TAG))
 
-ifdef METRICS_EXPORTER_ENABLED
```

Comment:
Please revert `METRICS_EXPORTER_ENABLED` and `EXPOSE_PUBLIC_SERVICE` variables. They are helpful during debugging.